app/sidekiq/evss/disability_compensation_form/upload_bdd_instructions.rb

```diff
@@ -89,21 +114,20 @@ def perform(submission_id)
       private
 
       def upload_bdd_instructions
-        client.upload(file_body, document_data)
+        if Flipper.enabled?(:disability_compensation_use_api_provider_for_bdd_instructions)
+          provider = self.class.api_upload_provider(submission)
+
+          upload_document = provider.generate_upload_document(BDD_INSTRUCTIONS_FILE_NAME)
+          provider.submit_upload_document(upload_document, file_body)
+        else
+          EVSS::DocumentsService.new(submission.auth_headers).upload(file_body, document_data)
```

Comment:
This is the existing job behavior, which calls the EVSS service directly. When this PR merges nothing should change. When we do enable `disability_compensation_use_api_provider_for_bdd_instructions` in production and start using the API providers, `disability_compensation_use_api_provider_for_bdd_instructions` will be a kill switch to revert to what this job was doing before.